.github/workflows/configlet.yml

```diff
@@ -13,7 +13,4 @@ jobs:
       run: bin/fetch-configlet
 
     - name: configlet lint
-      run: bin/configlet lint
-
-    - name: configlet fmt
-      run: bin/configlet fmt . && git diff --exit-code
```

Comment:
@ErikSchierboom there's no `fmt` or anything similar command on configlet.

Is this step still applicable?

Reply:
Nope. We don't have that (yet).